app/src/components/common/layout/header/index.tsx

```diff
@@ -284,6 +288,15 @@ const HeaderContainer: React.FC = (props: any) => {
             />
           )}
 
+          <HeaderButton>
+            {relay
```

Comment:
I just noticed this...After switching from xDai to mainnet....Xdai balance is not showing. This is not related to your pr but I will solve it on omn token bridge....

https://user-images.githubusercontent.com/33226956/116867966-385bec00-ac0e-11eb-9040-6ec4ac30deef.mov